src/Sylius/Component/Core/spec/Model/ChannelPricingSpec.php

```diff
@@ -49,4 +50,41 @@ function its_price_is_mutable()
         $this->setPrice(1000);
         $this->getPrice()->shouldReturn(1000);
     }
+
+    function it_does_not_have_original_price_by_default()
+    {
+        $this->getOriginalPrice()->shouldReturn(null);
+    }
+
+    function its_original_price_should_be_mutable()
+    {
+        $this->setOriginalPrice(2000);
+        $this->getOriginalPrice()->shouldReturn(2000);
+    }
+
+    function its_original_price_should_not_be_lower_than_price()
```

Comment:
Should we have it? I copied as it was from the PR where the feature was removed, and definitely the test is not correct